src/mailing/sender.js

```diff
@@ -43,5 +45,34 @@ async function sendTestEmail(to, username) {
   });
 }
 
-// Export the transport and the sendTestEmail function
-module.exports = { sendTestEmail };
+async function sendForgotPasswordEmail(toEmail, resetToken) {
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: process.env.EMAIL_USER,
+      pass: process.env.EMAIL_PASSWORD,
+    },
+  });
+
+  // Load and render the HTML template
+  const htmlTemplate = await ejs.renderFile(
+    path.join(__dirname, './templates/forgot_password/html.ejs'),
+    { resetLink: `http://localhost:8000/api/v1/reset-password/${resetToken}` }
+  );
+
+  const mailOptions = {
+    from: 'your@example.com',
```

Comment:
This email **must** match the user used in the `auth` otherwise when it won't send the email (at least this is the case for most email services) and you will see an SMTP error in the backend terminal logs.

```suggestion
    from: process.env.EMAIL_USERNAME,
```